source/extensions/quic_listeners/quiche/active_quic_listener.cc

```diff
@@ -42,6 +42,9 @@ ActiveQuicListener::ActiveQuicListener(
           &listener_config),
       dispatcher_(dispatcher), version_manager_(quic::CurrentSupportedVersions()),
       kernel_worker_routing_(kernel_worker_routing) {
+  // This flag fix a QUICHE issue which may crash Envoy during connection close.
```

Comment:
Should this be a TODO and/or link to a follow up issue? Do we plan on removing this later?

Reply:
quiche flags are like our runtime guards only they start false - it'll move to true by default and get removed over some number of import cycles.